tests/include.rs

```diff
@@ -55,6 +139,30 @@ fn should_include_array_u16_single() {
     assert!(included == expected, "included doesn't match expected file output");
 }
 
+#[test]
+fn should_include_array_u16le_single() {
+    let included = include_bytes!("tests/include.in" as [u16le; 24]);
+    let expected = read("tests/include.in").expect("To read source code");
+    let expected = unsafe {
+        slice::from_raw_parts(expected.as_ptr() as *const [u16; 24], included.len())
+    };
+    assert!(included == expected, "included doesn't match expected file output");
+}
+
+#[test]
+fn should_include_array_u16be_single() {
+    let included = include_bytes!("tests/include.in" as [u16be; 24]);
+    let mut expected = read("tests/include.in").expect("To read source code");
+    // swap bytes
+    for i in 0..expected.len() / 2 {
```

Comment:
In general it should be done on LE targets only, but I don't think I'd ever want to bother with setting up BE target test runner

Reply:
Yeah, for clarity sake it would be useful